st2common/st2common/models/utils/action_alias_utils.py

```diff
@@ -14,17 +14,33 @@
 # limitations under the License.
 
 import re
+import sys
+from sre_parse import (
```

Comment:
This looks like a CPython specific thing, but I could be wrong.

Is it also available in other implementations such as PyPy? In theory since it'a Python module it should work in PyPy as well, but you never know since it seems to be undocumented module from stdlib which is not really meant for public consumption so this could cause issues down the road.

Reply:
The `sre_parse` module does exist in PyPy, so this should also work there.

The internals of `re` are relatively well hidden, and the `parse` function changed between Python 2 and 3, but it otherwise looks stable and common between Python implementations.

Does Stackstorm test against PyPy in Travis or CircleCI?

Reply:
Thanks for clarifying.

Right now we only test with CPython because builds are already slow and running everything also on PyPy would take much longer, but we do plan to do it as some point in the future.